src/pages/Swap/index.tsx

```diff
@@ -350,6 +353,31 @@ export default function Swap({ history }: RouteComponentProps) {
 
   const priceImpactTooHigh = priceImpactSeverity > 3 && !isExpertMode
 
+  // arbitrum network alert
+  const [arbitrumAlertVisible, setArbitrumAlertVisible] = React.useState(false)
```

Comment:
not sure why you need state for this, just store whether it's dismissed in redux, and you should show it if they're on arbitrum and haven't dismissed it yet